.dockerignore

```diff
@@ -1,3 +1,18 @@
-node_modules
 .git
+.github
+.storybook
+base
+config_frontend
+coverage
 dist
+docs
+ingress
+node_modules
+overlays
+packages/components/dist
+packages/components/node_modules
+packages/utils/dist
```

Comment:
can we just ignore `packages` entirely so we don't need to worry about this if we add more frontend packages in future?

Reply:
I don't think we can ignore `packages/src` as we need that to build the frontend in the Dockerfile.

Reply:
oops, yeah that could be tough to do without the source 😆

Reply:
```
**/.git
**/dist
**/node_modules
```
maybe ?

Reply:
let's leave it as-is for now, looks like it doesn't follow the same format as .gitignore 😞

Reply:
https://docs.docker.com/engine/reference/builder/#dockerignore-file

> Beyond Go’s filepath.Match rules, Docker also supports a special wildcard string ** that matches any number of directories (including zero). For example, **/*.go will exclude all files that end with .go that are found in all directories, including the root of the build context.